src/assets/style.css

```diff
@@ -465,7 +462,7 @@ details[open] > summary {
   -webkit-user-select: all;
   user-select: all;
   background: none;
-  font-size: 50%;
+  overflow: hidden;
```

Comment:
I've also [accidentally] snuck in an improvement to the Markdown mention on the profile page, replacing the itsy-bitsy font with `overflow: hidden`.